database/factories/AnimeFactory.php

```diff
@@ -96,7 +98,10 @@ public function definition(): array
                     "name" => "Shounen",
                     "url" => "https://myanimelist.net/anime/genre/27/Shounen"
                 ]
-            ]
+            ],
+            "createdAt" => new UTCDateTime(),
+            "modifiedAt" => new UTCDateTime(),
+            "request_hash" => sprintf("request:%s:%s", "v4", sha1("http://localhost-test/v4/anime/" . $mal_id))
```

Comment:
Should this be dynamic?
`sha1("http://localhost-test/v4/anime/" . $mal_id)`
to
`sha1(env('APP_URL')."/v4/anime/" . $mal_id)`

Reply:
I wasn't sure about it. I don't know if there will be a test which would be relying on this. But I agree.